outofcore/include/pcl/outofcore/octree_base.h

```diff
@@ -612,24 +612,14 @@ namespace pcl
         checkExtension (const boost::filesystem::path& path_name);
 
 
-        /** \brief DEPRECATED - Flush all nodes' cache 
-         *  \deprecated this was moved to the octree_node class
-         */
+        /** \brief Flush all nodes' cache */
         void
         flushToDisk ();
 
-        /** \brief DEPRECATED - Flush all non leaf nodes' cache 
-         *  \deprecated
-         */
+        /** \brief Flush all non leaf nodes' cache */
         void
         flushToDiskLazy ();
 
-        /** \brief DEPRECATED - Flush empty nodes only 
-         *  \deprecated
-         */
-        void
-        DeAllocEmptyNodeCache ();
```

Comment:
Shall we keep this? Otherwise this PR will be tagged as breaking the ABI (since default arguments and function pointers don't work well)